src/functions.php

```diff
@@ -237,6 +237,12 @@ function get_exit_code(
     return $process->getExitCode() ?? 0;
 }
 
+function get_exit_code(...$args): int
+{
+    trigger_deprecation('jolicode/castor', '0.8', 'The "%s" function is deprecated, use "%s" instead.', __FUNCTION__, 'exit_code');
```

Comment:
nice!

```suggestion
    trigger_deprecation('jolicode/castor', '0.8', 'The "%s()" function is deprecated, use "exit_code()" instead.', __FUNCTION__);
```

same for other deprecation message

Reply:
done